.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Always nice to remove the unnecessary modules that live else where on the internet